pypff-stubs/pypff.pyi

```diff
@@ -0,0 +1,193 @@
+import datetime
+from typing import Any, Iterator, Iterable, Optional
+"""
+The stubs file for pypff, a python wrapper for libpff
+
+We base our understanding of the structure of each class through reading the
+class definitions in [1]. One peculiarity of the pypff library are the large amount
+of unnecessary "getter" methods that fetch class instance variables. 
+
+[1] https://github.com/libyal/libpff/tree/main/pypff
+"""
+
+def file() -> File:
+    """
+    The constructor for th pypff.File object.
+
+    We cannot use __init__ to define this constructor, since the method call does not follow Python
+    naming convention
+    """
+
+class File:
+    ### Instance variables
+    # Unused instance variables
+    # size
+    # content_type
+    # encryption_type
+    # ascii_codepage
+    # root_item
+    # message_store
+    # name_to_id_map
```

Comment:
It seems like it's almost as much work to maintain a comment with the unused members as it is to just document the members? I'd prefer either adding the members, or ignoring them entirely (although just adding the members would allow us to use them in our python code without having to package a new version of our pypff fork).

Reply:
9ec1fd476cfdc42181cc4045ba978708caa720c7